ql/ql/src/queries/style/GetAPrimaryQlClassConsistency.ql

```diff
@@ -10,18 +10,29 @@
 
 import ql
 
-from ClassPredicate pred, String constant
+/**
+ * Gets a string that is allowed to be returned from the `getAPrimaryQlClass` implementation
+ * in class `c`.
+ *
+ * For most languages this is just the name of `c`, but QL for Swift implements
+ * the `getAPrimaryQlClass` predicate for a class `Foo` in another class `FooBase`.
+ */
+string getAnAllowedResultForClass(Class c) {
+  exists(string className | c.getName() = className | result = className + ["", "Base"])
```

Comment:
```suggestion
  result = c.getName() + ["", "Base"]
```
?

Reply:
Indeed! That was me copy-pasting without thinking 😂. Thanks!